app/api/__init__.py

```diff
@@ -15,7 +16,14 @@
 
 # users
 api.route(UserList, 'user_list', '/users')
-api.route(UserDetail, 'user_detail', '/users/<int:id>')
+api.route(UserDetail, 'user_detail', '/users/<int:id>', '/notifications/<int:notification_id>/user')
+api.route(UserRelationship, 'user_notification', '/users/<int:id>/relationships/notifications')
+
+#notifications
+api.route(NotificationList, 'notification_list', '/notifications', '/users/<int:id>/notifications')
+api.route(NotificationDetail, 'notification_detail', '/notifications/<int:id>', 'users/<int:id>/notifications')
```

Comment:
I think this shouldn't be here. But please check if the endpoints follow this relationship: https://github.com/fossasia/open-event-orga-server/pull/3704/files#diff-b0eafe947274d1253c0a6f754259776aR46 @poush